### PR TITLE
Adding support for multiple attributes for instance query

### PR DESCRIPTION
**Pull request by vi-patel**
## What does this implement/fix? Explain your changes
Adding support to query for multiple attributes:
miqcli --verbose instances query --attr hardware --attr floating_ip --attr active

Previously, you could only pass a single attribute, but there may be cases where it would be beneficial to see more attributes.

api call updated:

previously, attr was a str
client.collection.query(inst_name="<vm_name>", attr="floating_ip")

updated to now be a tuple:
client.collection.query(inst_name="<vm_name>", attr=("floating_ip",))

 * update requires update to the example scripts
 * while working on the update, fixed an issue w/ displaying all instances w/attributes set (header was previously set for each instance instead of being displayed once).

## Does this close any currently open issues?
no, this is an enhancement

### CODE REVIEW

**Review by smatula — APPROVED** [review]
LGTM

File: example/osp_provision_example.py [hunk 1]
```diff
@@ -98,10 +98,13 @@
 
 vm_name = payload_data["vm_name"]
 # 7. Report the floating ip address back to the user
+# need an extra sleep to make sure floating ip is added to
+# the instance after provisioning
+sleep(10)
```
Comment (ryankwilliams) [inline example/osp_provision_example.py:103]: Just a comment: I think it is okay for this example script to have the sleep to ensure the floating IP is attached to the instance. In the future when a script is provided to handle all of this, this area should be enhanced that when doing the query if it is unable to get the floating IP, have it do some sort of retry. Maybe give X amount of retries before failing. I just am thinking of network latency or connectivity issues could cause 10 seconds maybe to not be enough time.

**Review by ryankwilliams — APPROVED** [review]
LGTM

**Review by tvieira — APPROVED** [review]
LGTM

**Pull request merged**